src/lucky_record/callbacks.cr

```diff
@@ -1,3 +1,34 @@
+# Callbacks will trigger before and after the `create`, `save`, or `update` method is called on a form.
+#
+# To define a callback, you assign a method to one of the callback groups:
+#
+# ```crystal
+# class UserForm < User::BaseForm
+#   before_create set_token
```

Comment:
The problem with `set_token` is that something like that should almost always happen in `prepare`

How about something like `before_create log_user_creation`. Probably not the best example, but that's probably ok since *before* should rarely be used.

It might also be worth mentioning that *before* is rarely used and that validations and setting values should usually happen in `prepare`